packages/google_sign_in/google_sign_in_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.1
```

Comment:
This is just a bugfix, not an API change, so it should be 0.12.0+3

Reply:
Thank you, updated!